apps/docs/app/api/entelligence-history/route.ts

```
@@ -0,0 +1,11 @@
+export const POST = async (req: Request) => {
+  await fetch(process.env["ENTELLIGENCE_HISTORY_API_URL"]!, {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)